projects/storefrontlib/src/feature-toggles.ts

```diff
@@ -8,6 +8,5 @@ export interface FeatureToggles {
     level?: string;
     consignmentTracking?: boolean;
     saveForLater?: boolean;
-    cmsPageLoadOnce?: boolean;
```

Comment:
Document removing that flag and how to replace the behavior with new config in migration guide.

Reply:
👍